tests/fixtures/organisations.py

```diff
@@ -867,7 +889,8 @@ def circulation_policies(
         circ_policy_short_martigny,
         circ_policy_temp_martigny,
         circ_policy_ebooks_martigny,
-        circ_policy_ebooks_sion
+        circ_policy_ebooks_sion,
+        circ_policy_less_than_one_day_martigny
```

Comment:
If I understand correctly, you wanted to remove `circ_policy_less_than_one_day_martigny` from  `circulation_policies`. If tests are passing then should be OK.

Reply:
Yes, I removed this cipo from the list, as it is not supposed to follow a circulation scenario but a simpler case. I test it in a dedicated file.